### ISSUE TITLE
'Info' icons containing tooltip information is not accessible under 'New collection' blade.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1014]
https://user-images.githubusercontent.com/8519407/132721742-8dd88d35-48f9-4567-a7ec-c6078e31b5b0.mp4

Note: User credentials should NOT be included in the bug. 
Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Add Collection" Menu item/Button and select it.
Navigate through all the controls under add Collection/New collection blade.
Navigate to all fields containing 'info' icons like radio controls and edit fields.
Press F1 shortcut key or "Shift+ F1" key to access 'info' icons tooltip.
Verify the issue.

Actual Result:
'Info' icons containing tooltip information is not accessible under 'New collection' blade.

Expected Result:
'Info' icons containing tooltip information which are accessible using mouse hovering under 'New collection' blade should also be accessible using keyboard.

Note:
Similar issue exists with all info icon controls present under "New Collection" blade and 'New Database' blade. Screen reader is not reading tooltip information as well.
This issue also exists for similar controls present under 'New Graph' blade of Data explorer screen for Created cosmos DB  after selecting "Gremlin(graph)" option.
This issue exists throughout the application for all info icon controls of Data Explorer page.

**Comment by ato9000** [comment]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1216103
